lib/linguist/languages.yml

```diff
@@ -2346,6 +2346,8 @@ JSON:
   searchable: false
   extensions:
   - ".json"
+  - ".4DForm"
```

Comment:
Could you add a sample file in `samples/JSON/` for this extension?

Reply:
You can find a sample here : https://github.com/vdelachaux/4DPop-Git/Project/Sources/Forms/GIT/form.4DForm